docs/BrandColors.stories.tsx

```diff
@@ -22,6 +26,7 @@ type Story = StoryObj<typeof ColorSwatchGroup>;
 export const Figma: Story = {
   render: () => {
     const { brandColor } = useJsonColor();
+    console.log(brandColor);
```

Comment:
Let's retain console logs in the stories while we are working on these updates I find it's quite helpful. Maybe we can change the console logs to
```suggestion
    console.log('brandColor JSON', brandColor);
```